### PR TITLE
OpenHPC 2

### PR DESCRIPTION
**Pull request by oneswig**
Review after merge of #35 
Building on Steve's work to add OpenHPC 2 support

### CODE REVIEW

**Comment by brtknr** [comment]
@oneswig can you please rebase on current master?

File: README.md [hunk 1]
```diff
@@ -133,3 +133,13 @@ To drain nodes, for example, before scaling down the cluster to 6 nodes:
...
 
+
+CentOS 8 and OpenHPC 2
+----------------------
+
+To deploy OpenHPC 2 on CentOS 8, you must first enable the CentOS PowerTools repo
+(this ships as standard, but disabled).  To enable PowerTools:
+
+```
+sudo dnf config-manager --set-enabled PowerTools
```
Comment (brtknr) [inline README.md:144]: We really ought to do this via this role dont you think?

File: tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+- name: Gather package facts
+  package_facts:
+    manager: rpm
+- assert:
+    that: "'ohpc-release' in ansible_facts.packages"
+    quiet: yes
+    fail_msg: |
+      "The OpenHPC distro package should be installed in the image.
+       Please see http://openhpc.community/downloads/"
+- debug:
```
Comment (sjpb) [inline tasks/main.yml:13]: Remove debug task?

File: README.md (hunk 1 above)
Comment (oneswig) [reply README.md:144]: I think so too but let's do changes to the deployed repos as part of #6 - which shouldn't be too tricky to add in due course.

File: vars/ohpc-1.3 [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+# OpenHPC 1.3 on CentOS 7
```
Comment (sjpb) [inline vars/ohpc-1.3:2]: I've just run a version of this branch on a base Centos**7** image with all the "@" removed and it worked fine. So maybe we don't actually need to differentiate between v1 and v2?

Comment (sjpb) [reply vars/ohpc-1.3:2]: However I like the package list setup - much tidier/faster than it was before.

Comment (oneswig) [reply vars/ohpc-1.3:2]: Good to know.  We still need the differences, but only for munge now.

File: tasks/main.yml (hunk 2 above)
Comment (oneswig) [reply tasks/main.yml:13]: Oops, thanks, will do.

**Comment by sjpb** [comment]
Other than the above looks good to me - have tested on alaska VMs and worked fine.

**Pull request merged**